lua/gk.lua

```diff
@@ -26,6 +26,9 @@ return function (net_conf, sol_conf, gk_lcores)
 	-- Scan the whole flow table in 10 minutes.
 	gk_conf.flow_table_full_scan_ms = 10 * 60 * 1000
 
+	-- The maximum number of packets to retrieve/transmit.
+	gk_conf.gk_max_pkt_burst = 32
```

Comment:
I just realized that we must have this field per interface because each interface may be a different kind of bonding. My suggestion is to drop field `gk_max_pkt_burst` and add fields `front_max_pkt_burst` and `back_max_pkt_burst`. Notice that this is needed for the following blocks: GK, LLS, and CPS.

Instead of having a constant to set up these parameters for GK and GT, use `net_conf` to find out the number of ports in the bonding and obtain the ideal number. For example: `math.max(8, front_num_ports * 2)`.